CHANGELOG.md

```diff
@@ -59,6 +59,7 @@
 * [ENHANCEMENT] Dashboards: Add panels to the `Mimir / Tenants` and `Mimir / Top Tenants` dashboards showing the rate of gateway requests. #10978
 * [ENHANCEMENT] Alerts: Improve `MimirIngesterFailsToProcessRecordsFromKafka` to not fire during forced TSDB head compaction. #11006
 * [ENHANCEMENT] Alerts: Add alerts for invalid cluster validation labels. #11255 #11282
+* [ENHANCEMENT] Alerts: Enhance the `MimirRolloutStuck` alert, so it checks whether rollout groups as a whole (and not spread across instances) are changing or stuck. #11288
```

Comment:
I would say this is a bugfix: the alert has a false positive when some of the kubernetes metrics are exposed by different ksm-metrics instances.

Reply:
OK, will change it.

Reply:
Done.